pkg/chartutil/create.go

```diff
@@ -147,16 +147,13 @@ service:
 
 ingress:
   enabled: false
+  # className: nginx
```

Comment:
what's the reason behind this comment?

Reply:
it's not really a comment; it will be emitted to the values.yaml as a hint that this value is configurable

if not set, kubernetes will use the default ingress class - which might be different to nginx (e.g. contour, ambassador etc .etc.)

you see something similar on the resource hints

Reply:
This is a new optional field allowing to customize the ingress class name. It's also backward compatible with older ingress versions, so you can use it instead of the annotation: on older Kubernetes versions it will fall back to the annotation.